h2o-core/src/main/java/water/fvec/Frame.java

```diff
@@ -1578,4 +1578,14 @@ public int getCurrentRowSize() throws IOException {
   /** Sort rows of a frame, using the set of columns as keys.
    *  @return Copy of frame, sorted */
   public Frame sort( int[] cols ) { return Merge.sort(this,cols); }
+
+    public Boolean hasCategoricalColumn() {
```

Comment:
1. What's up with the indentation here? we use 2-space indentation in Java code (and Google Java style guide suggests the same).
2. The return type is `boolean` not `Boolean`.
3. This method should not exist in the `Frame`. There are literally hundreds of operations that may be invoked with a frame, and if we start putting all of them into the Frame class, it will eventually become a monstrosity of epic proportions.